File: simple_geo3x3.kt
fun main(args: Array<String>) { 
  var code = Geo3x3.encode(35.65858, 139.745433, 14)
  println(code)
  var res = Geo3x3.decode("E3793653391822")
  println("${res[0]} ${res[1]} ${res[2]} ${res[3]}")
}

File: simple_geo3x3.rb
require "./geo3x3"

## get Geo3x3 code from latitude / longitude / level
code = Geo3x3.encode(35.65858, 139.745433, 14)
p code # "E3793653391822"

## get location from Geo3x3 code
pos = Geo3x3.decode('E3793653391822')
p pos # [35.658633790016204, 139.74546563023935, 14, 0.00011290058538953522]
